app/controllers/application_controller/ci_processing.rb

```diff
@@ -1067,7 +1067,7 @@ def set_memory_cpu
     #set memory to nil if multiple items were selected with different mem_cpu values
     @reconfigureitems.each_with_index do |vm, i|
       @temp_memory = i == 0 || @temp_memory == vm.mem_cpu ? vm.mem_cpu : nil
-      @temp_cpu_count = i == 0 || @temp_cpu_count == vm.num_cpu ? vm.num_cpu : nil
+      @temp_cpu_count = i == 0 || @temp_cpu_count == vm.logical_cpus ? vm.logical_cpus : nil
```

Comment:
This is very unclear without parentheses.  Even with parentheses, the i == 0 case should probably be pulled our for clarity as follows:

```ruby
@reconfigureitems.each_with_index do |vm, i|
  if i == 0
    @temp_memory = vm.mem_cpu
    @temp_cpu_count = vm.logical_cpus
    @temp_cores_per_socket_count = vm.cores_per_socket
  else
    @temp_memory = nil unless @temp_memory == vm.mem_cpu
    @temp_cpu_count = nil unless @temp_cpu_count == vm.logical_cpus
    @temp_cores_per_socket_count = nil unless @temp_cores_per_socket_count == vm.cores_per_socket
  end
end
```